test/e2e/logs_test.go

```diff
@@ -173,6 +173,24 @@ var _ = Describe("Podman logs", func() {
 		Expect(string(out)).To(ContainSubstring("alpine"))
 	})
 
+	It("podman journald logs for container name", func() {
+		Skip("need to verify images have correct packages for journald")
```

Comment:
You test is always skipped?

Reply:
all of the logs tests are always skipped

Reply:
they're actually skipped in two different spots :smile: